### PR TITLE
Use double quotes "" to allow substitution

### PR DESCRIPTION
**Pull request by wantehchang**
Also remove an extraneous space in tests/oss-fuzz/build.sh.

### CODE REVIEW

File: .github/workflows/ci-unix-static-sanitized.yml [hunk 1]
```diff
@@ -49,7 +49,7 @@ jobs:
         if: steps.cache-ext.outputs.cache-hit != 'true'
         working-directory: ./ext
         run: >
-          sed -i -e 's/meson setup \(.*\) \.\./meson setup \1 '"-Db_sanitize=${{ matrix.sanitizer }} -Db_lundef=false"' ../g' dav1d.cmd
+          sed -i -e 's/meson setup \(.*\) \.\./meson setup \1 -Db_sanitize=${{ matrix.sanitizer }} -Db_lundef=false ../g' dav1d.cmd
```
Comment (wantehchang) [inline .github/workflows/ci-unix-static-sanitized.yml:52]: James pointed out that the `-i` option is not portable. I did not address that issue in this pull request.

The `-i` option is not in the Single Unix specification:
https://pubs.opengroup.org/onlinepubs/9699919799/utilities/sed.html

And it is different between Linux and macOS.

Linux:

```
  -i[SUFFIX], --in-place[=SUFFIX]
                 edit files in place (makes backup if SUFFIX supplied)
```

macOS:

```
     -i extension
             Edit files in-place similarly to -I, but treat each file
             independently from other files.  In particular, line numbers in
             each file start at 1, the “$” address matches the last line of
             the current file, and address ranges are limited to the current
             file.  (See Sed Addresses.) The net result is as though each file
             were edited by a separate sed instance.
```

Comment (y-guyon) [reply .github/workflows/ci-unix-static-sanitized.yml:52]: Yes, I had to add `-e` to make it work on macos.

**Review by y-guyon — APPROVED** [review]

File: .github/workflows/ci-unix-static-sanitized.yml (hunk 1 above)
Comment (wantehchang) [reply .github/workflows/ci-unix-static-sanitized.yml:52]: Thanks for the confirmation. James belives (and I agree) that on macOS, `sed -i -e script` is parsed as `sed`, `-i -e`, `script`. So `-e` is the argument to the `-i` option, i.e., `-e` is the extension of the backup file before `sed` edits the file in place.

File: tests/oss-fuzz/build.sh [hunk 2]
```diff
@@ -49,7 +49,7 @@ if [ "$SANITIZER" == "memory" ]
 then
   DAV1D_EXTRA_FLAGS="${DAV1D_EXTRA_FLAGS} -Denable_asm=false"
 fi
-sed -i 's/meson setup \(.*\) \.\./meson setup \1 '"${DAV1D_EXTRA_FLAGS}"' ../g' ./ext/dav1d.cmd
+sed -i "s/meson setup \(.*\) \.\./meson setup \1${DAV1D_EXTRA_FLAGS} ../g" ./ext/dav1d.cmd
```
[the remark this answers is not in the document]
Comment (vrabaud) [reply tests/oss-fuzz/build.sh:52]: That's fine

**Pull request merged**